exchanges/kraken/kraken_wrapper.go

```diff
@@ -167,9 +168,9 @@ func (k *Kraken) Run() {
 	}
 
 	forceUpdate := false
-	if !common.StringDataContains(k.GetEnabledPairs(asset.Spot).Strings(), "-") ||
-		!common.StringDataContains(k.GetAvailablePairs(asset.Spot).Strings(), "-") {
-		enabledPairs := currency.NewPairsFromStrings([]string{"XBT-USD"})
+	if !common.StringDataContains(k.GetEnabledPairs(asset.Spot).Strings(), k.CurrencyPairs.ConfigFormat.Delimiter) ||
+		!common.StringDataContains(k.GetAvailablePairs(asset.Spot).Strings(), k.CurrencyPairs.ConfigFormat.Delimiter) {
+		enabledPairs := currency.NewPairsFromStrings([]string{fmt.Sprintf("BTC%vUSD", k.CurrencyPairs.ConfigFormat.Delimiter)})
```

Comment:
This will always execute as the delimiter is empty and thus won't be found, so it will reset the enabled pair to BTCUSD, but since this isn't an available pair, will be reset again to a random avail pair